templates/article.liquid

```diff
@@ -128,15 +128,12 @@
           {% endif %}
 
           {% comment %}
-            Comment submission form
+            #comment-form is required, it is used as an anchor link by Shopify
+            when an error occurs during submission.
           {% endcomment %}
-          <div class="form-vertical">
+          <div id="comment-form" class="form-vertical">
             {% form 'new_comment', article %}
```

Comment:
this will take care of generating the `id="comment_form"`, as explained in the note above :)